.gitignore

```diff
@@ -3,6 +3,6 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
```

Comment:
The `.venv/` entry is correctly included to ignore virtual environment directories. However, ensure that this matches the actual directory name used in your project. If your virtual environment directory has a different name, update this entry accordingly.